src/main/java/org/broadinstitute/hellbender/tools/exome/FilterByOrientationBias.java

```diff
@@ -25,9 +26,44 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+/**
+ * Additionally filter Mutect2 somatic variant calls for sequence-context dependent artifacts.
+ *
+ * <p>
+ *     This tool is complementary to {@link org.broadinstitute.hellbender.tools.walkers.mutect.FilterMutectCalls}.
+ *     The tool requires the pre-adapter detailed metrics calculated by Picard CollectSequencingArtifactMetrics.
+ *     Specify the sequence context(s) to consider for filtering with the --artifactModes argument.
```

Comment:
>Specify the sequence contexts(). . .

This isn't quite right.  Rather it specifies what types of substitutions eg G --> T to consider.  For every substitution, the Picard detailed metrics measure orientation bias at all 3-base contexts, but the user can't control that.  Note also that if you specify a substitution the tool filters both it and its reverse complement eg G --> T *and* C --> A.  Only one substitution in a reverse complement pair should be specified.